noir-projects/noir-protocol-circuits/crates/types/src/hash.nr

```diff
@@ -45,6 +45,8 @@ pub fn compute_note_hash_nonce(first_nullifier: Field, note_hash_index: u32) ->
     )
 }
 
+// TODO(benesjan): some of these functions seem to be duplicate to the ones in
+// noir-projects/aztec-nr/aztec/src/note/utils.nr NUKE!
```

Comment:
Will address this one in [follow-up PR](https://github.com/AztecProtocol/aztec-packages/pull/7132).